terraform/spot_fleet_request/iam.tf

```diff
@@ -1,36 +0,0 @@
-data "aws_iam_policy_document" "assume_spot_fleet_role" {
```

Comment:
Removing these for now, but they are still valuable if we wish to use _only_ spot fleet requests in the future.